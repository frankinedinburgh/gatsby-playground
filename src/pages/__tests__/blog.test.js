import React from "react"
import renderer from "react-test-renderer"

import Blog from "../blog"

describe("Blog", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Blog />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})