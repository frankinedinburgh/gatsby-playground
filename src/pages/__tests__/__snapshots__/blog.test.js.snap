// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blog renders correctly 1`] = `
<div>
  <h1>
    Blog
  </h1>
  <p>
    Posts will show up here later on.
  </p>
</div>
`;
